src/Countries/Canada.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+
+class Canada extends Country
+{
+    public function countryCode(): string
+    {
+        return 'ca';
+    }
+
+    // These 5 statutory holidays are the only constant among all provinces
```

Comment:
There is documentation on how to add province based holidays: https://github.com/spatie/holidays?tab=readme-ov-file#adding-a-new-country